controller/src/main/java/io/pravega/controller/fault/ControllerClusterListener.java

```diff
@@ -74,18 +88,6 @@ protected void startUp() throws InterruptedException, Exception {
             log.info("Registering host {} with controller cluster", host);
             cluster.registerHost(host);
 
-            if (eventProcessorsOpt.isPresent()) {
```

Comment:
Moving line 129 `taskSweeper.sweepOrphanedTasks(processes); ` before this block that waits for eventProcessors and txnSweeper to get ready would be a simple fix the issue, isn't it?